Rely on core_optional_deps in the root CMakeLists.txt instead to find the packages.

diff --git a/cmake/modules/FindFFMPEG.cmake b/cmake/modules/FindFFMPEG.cmake
index 407859b71d..2dc91d41b2 100644
--- a/cmake/modules/FindFFMPEG.cmake
+++ b/cmake/modules/FindFFMPEG.cmake
@@ -115,18 +115,10 @@ macro(buildFFMPEG)
   else()
 
     if(CORE_SYSTEM_NAME STREQUAL linux OR CORE_SYSTEM_NAME STREQUAL freebsd)
-      if("VAAPI" IN_LIST PLATFORM_OPTIONAL_DEPS)
-        find_package(VAAPI)
-      endif()
-
       if(TARGET ${APP_NAME_LC}::VAAPI)
         set(FFMPEG_VAAPI TRUE)
       endif()
 
-      if("VDPAU" IN_LIST PLATFORM_OPTIONAL_DEPS)
-        find_package(VDPAU)
-      endif()
-
       if(TARGET ${APP_NAME_LC}::VDPAU)
         set(FFMPEG_VDPAU TRUE)
       endif()

